integrations/extensions/starter-kits/testitall/mock-server/server/controllers/test-controller.js

```diff
@@ -84,12 +174,6 @@ export function contextAlmostTooLargeTest(req, res) {
   return res.status(200).send(response);
 }
 
-export function successfulPostWithNonJSONResponse(req, res) {
```

Comment:
Not sure why this got deleted in your PR?

Reply:
Interesting. I rebased before pushing the changes. Is it possible that it got deleted during the rebase? When I ran `difftool`, I only saw the new files/functions. Anyway, I'll compare it to the latest version in `master` and fix it.